src/ck-ldb/LBManager.C

```diff
@@ -518,7 +527,7 @@ void LBManager::init(void)
   if (manualOn) TurnManualLBOn();
 #endif
 
-  if (_lb_args.lbperiod() != -1.0)  // check if user set LBPeriod - fix later
+  if (_lb_args.lbperiod() != -1.0)
```

Comment:
Would this be safer as `if (_lb_args.lbperiod() > 0.0)`?

Reply:
The intention was just to see if they had changed it from the default, but I think your suggestion is better, we should make sure it's positive, too. I'll change it.